src/script.js

```diff
@@ -3,13 +3,36 @@ import './style.css';
 const addButton = document.querySelector('.addButton');
 const inputValue = document.querySelector('.input');
 const container = document.querySelector('.container');
+const tasks = [
+  {
+    description: 'Buy groceries',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Take out the trash',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Walk the dog',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Do laundry',
+    completed: false,
+    index: 3,
+  },
+];
```

Comment:
- [ ] kindly consider removing this for this milestone as requested in the project requirements and this will help you to full-fill the below requirements 
- > Remove all hardcoded items from the tasks array.